lib/administration/ticket/include/irods/ticket_administration.hpp

```diff
@@ -0,0 +1,577 @@
+        template <typename TicketConstraint>
+        void remove_ticket_constraint(admin_tag,
+                                      RxComm& conn,
+                                      const std::string_view ticket_name,
+                                      const TicketConstraint& constraint)
+        {
+            if constexpr (std::is_same_v<user_constraint, TicketConstraint>) {
+                detail::execute_ticket_operation(conn,
+                                                 "mod",
+                                                 ticket_name,
+                                                 "remove",
+                                                 "user",
+                                                 constraint.value,
+                                                 "",
+                                                 true);
+            }
+            else if constexpr (std::is_same_v<group_constraint, TicketConstraint>) {
+                detail::execute_ticket_operation(conn,
+                                                 "mod",
+                                                 ticket_name,
+                                                 "remove",
+                                                 "group",
+                                                 constraint.value,
+                                                 "",
+                                                 true);
+            }
+            else if constexpr (std::is_same_v<host_constraint, TicketConstraint>) {
+                detail::execute_ticket_operation(conn,
+                                                 "mod",
+                                                 ticket_name,
+                                                 "remove",
+                                                 "host",
+                                                 constraint.value,
+                                                 "",
+                                                 true);
+            }
+            else if constexpr (std::is_same_v<use_count_constraint, TicketConstraint>) {
+                detail::execute_ticket_operation(conn, "mod", ticket_name, "uses", "0", "", "", true);
+            }
+            else if constexpr (std::is_same_v<n_writes_to_data_object_constraint, TicketConstraint>) {
+                detail::execute_ticket_operation(conn, "mod", ticket_name, "write-file", "0", "", "", true);
+            }
+            else if constexpr (std::is_same_v<n_write_bytes_constraint, TicketConstraint>) {
+                detail::execute_ticket_operation(conn, "mod", ticket_name, "write-bytes", "0", "", "", true);
+            }
+            else {
+                throw std::invalid_argument("Invalid constraint type.");
+            }
+        }
+
+        /// \brief Remove the ticket constraint from the ticket using admin privilege
+        ///
+        /// \tparam TicketConstraint The template struct to indicate the type of constraint
```

Comment:
Add an empty line after this line so that the `\param` directive is more noticeable.